Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -88,8 +193,74 @@ public static int[] InsertLast(int[] array, int number)
         /// <returns>A new array with element inserted at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] InsertAt(int[] array, int number, int index)
```

Comment:
I don't really understand why you call the "same" function twice with and without "!" in the part below.
If you check for null or empty right after this line without calling a new function this will be fine and much more readable and understandable.

Reply:
sorry. my bad. One of the two needed another name